crates/sui-core/src/authority_aggregator.rs

```diff
@@ -1281,21 +1299,25 @@ where
 
             // Safe to unwrap because we know that there is at least one entry in the map
             // from the check above.
-            let (_most_staked_effects_digest, (_, most_staked_effects_digest_stake)) =
-                non_quorum_effects
-                    .iter()
-                    .max_by_key(|&(_, (_, stake))| stake)
-                    .unwrap();
+            // let (_most_staked_effects_digest, (_, most_staked_effects_digest_stake)) =
+            //     non_quorum_effects
+            //         .iter()
+            //         .max_by_key(|&(_, (_, stake))| stake)
+            //         .unwrap();
             // We check if we have enough retryable stake to get quorum for the most staked
             // effects digest, otherwise it indicates we have violated safety assumptions
             // or we have forked.
-            if most_staked_effects_digest_stake + self.get_retryable_stake(&state)
-                < self.committee.quorum_threshold()
-            {
-                panic!(
-                    "We have violated our safety assumption or there is a fork. Tx: {tx_digest:?}. Non-quorum effects: {non_quorum_effects:?}."
-                );
-            }
+            // if most_staked_effects_digest_stake + self.get_retryable_stake(&state)
+            //     < self.committee.quorum_threshold()
+            // {
+            //     println!("self.committee.quorum_threshold():{:?}", self.committee.quorum_threshold());
+            //     println!("self.get_retryable_stake(&state):{:?}", self.get_retryable_stake(&state));
+
+            //     println!("most_staked_effects_digest_stake:{:?}", most_staked_effects_digest_stake);
+            //     panic!(
+            //         "We have violated our safety assumption or there is a fork. Tx: {tx_digest:?}. Non-quorum effects: {non_quorum_effects:?}."
+            //     );
+            // }
```

Comment:
cc @arun-koshy I know that you added a panic here to indicate fork. i removed because i think in the case of an aggregated sig fails to verify and we fallback to the individual sig verify, the ones that failed are non retryable stake.  this is legitimate error that should be returned from L1308. take 4 validators as an example, 2 of them are byzantine, even though `most_staked_effects_digest_stake` is 2, the retryable stake here is 0 and this is below the quorum threshold. 

do you see anything im missing here? or is there a better way to differentiate 1) individual sig fallback adding up the nonretryable stake + do not panic but just returns error vs 2) what you have here to describe a fork that the validator should panic instead of returning error